Framework/PCEditorManager.m

```diff
@@ -206,6 +206,7 @@ - (void)loadPreferences:(NSNotification *)aNotification
           if ([[NSWorkspace sharedWorkspace] openFile: filePath])
             return nil;
         }
+      return nil;
```

Comment:
is this still needed given my newer changes in the external editor check? I think not, I do not have .gorm bundle issues anymore

Reply:
I didn't use external editor. I tried to test ProjectCenter as a newcomer.